conjure-java-core/src/main/java/com/palantir/conjure/java/types/MethodSpecs.java

```diff
@@ -104,19 +105,15 @@ private static CodeBlock getHashInput(Collection<FieldSpec> fields) {
                 .collect(joining(CodeBlock.of(", "))));
     }
 
-    public static MethodSpec createToString(String thisClassName, Collection<FieldName> fieldNames) {
+    public static MethodSpec createToString(String thisClassName, List<FieldName> fieldNames) {
         CodeBlock returnStatement = fieldNames.isEmpty()
                 ? CodeBlock.builder().addStatement("return $S", thisClassName + "{}").build()
                 : CodeBlock.builder()
```

Comment:
it would be nicer for `toStringConcatenation` to return a CodeBlock than to require maintaining across two methods...and easier to write a test for it.